packages/iov-tendermint-rpc/package.json

```diff
@@ -46,5 +46,8 @@
     "readonly-date": "^1.0.0",
     "type-tagger": "^1.0.0",
     "xstream": "^11.10.0"
+  },
+  "devDependencies": {
+    "@iov/utils": "^2.0.0-alpha.7"
```

Comment:
Good fix. This is used in `packages/iov-tendermint-rpc/src/client.spec.ts`: `import { sleep } from "@iov/utils";` but the dependency was missing.

Reply:
I didn't like sneaking it in with the VoteOptions stuff, but must press forward.

Reply:
You could have afforded a second commit at least ;)